tools/building.py

```diff
@@ -324,6 +324,9 @@ def lld_flags_for_executable(external_symbols):
       cmd.append('--growable-table')
 
   if not settings.SIDE_MODULE:
+    if settings.STACK_OVERFLOW_CHECK >= 2:
```

Comment:
A comment here would be good.   Mention how binaryen will strip the exported globals anyway?   Maybe a TODO to find a better solution than simply disabling the checks.